lte/gateway/docker/ghz/Dockerfile

```diff
@@ -25,10 +34,19 @@ FROM ${REPO_LOCATION}/agw_gateway_python as python_builder
 ARG GHZ_REPO=https://github.com/bojand/ghz
 
 RUN apt-get update && apt-get install -y \
+  curl \
   git \
-  golang \
   build-essential
 
+# Install golang 1.18
+WORKDIR /usr/local
+ARG GOLANG_VERSION="1.18"
+RUN GO_TARBALL="go${GOLANG_VERSION}.linux-amd64.tar.gz" \
+ && curl https://artifactory.magmacore.org/artifactory/generic/${GO_TARBALL} --remote-name --location \
+ && tar -xzf ${GO_TARBALL} \
+ && ln -s /usr/local/go/bin/go /usr/local/bin/go \
+ && rm ${GO_TARBALL}
+
```

Comment:
Duplicate of above line 15-23?

Reply:
These are different FROM-sections. You have a good idea to make this DRY?

Reply:
Should have thought that this is a multistage build, but this was not visible in Github on first sight (due to hidden lines).
To avoid this duplication one would need to rethink the whole building process here and give the file a more general cleanup, which is out of scope in this bugfix.

Considering that, this is the appropriate bugfix for the problem at hand.